ui/components/PromptComponent.js

```diff
@@ -89,9 +94,20 @@ class PromptComponent extends React.Component {
     this.setState({ show : false });
   };
 
+  handleCheckboxChange = () => {
+    this.setState((prevState) => ({
+      isChecked : !prevState.isChecked,
+    }));
+  };
+
+  getCheckboxState = () => {
```

Comment:
What is the use case of this function?

Reply:
This method function is required to get state of checkbox through ref, although I'm open for suggestion if you have